Library/Format.swift

```diff
@@ -114,14 +117,15 @@ public enum Format {
    - returns: A formatted string.
    */
   public static func date(secondsInUTC seconds: TimeInterval,
-                          dateFormat: String) -> String {
+                          dateFormat: String,
+                          timeZone: TimeZone = AppEnvironment.current.calendar.timeZone) -> String {
```

Comment:
i think you should be consistent and use the same signature that you use for the above `date` method. that is, make the argument of type `TimeZone?` and coalesce in the `formatter` declaration below.